src/UI/WelcomeView.re

```diff
@@ -13,84 +13,120 @@ open Oni_Core;
 module Model = Oni_Model;
 
 module Styles = {
-  let container = (~theme: Theme.t) => Style.[
-    backgroundColor(theme.background),
-    color(theme.foreground),
-    flexGrow(1),
-    flexDirection(`Column),
-    justifyContent(`Center),
-    alignItems(`Center)
-  ];
+  let container = (~theme: Theme.t) =>
+    Style.[
+      backgroundColor(theme.background),
+      color(theme.foreground),
+      flexGrow(1),
+      flexDirection(`Column),
+      justifyContent(`Center),
+      alignItems(`Center),
+    ];
 
-  let titleText = (~theme: Theme.t, ~font: UiFont.t) => Style.[
+  let titleText = (~theme: Theme.t, ~font: UiFont.t) =>
+    Style.[
       fontFamily(font.fontFile),
       fontSize(20),
       backgroundColor(theme.background),
       color(theme.foreground),
-      marginTop(-40)
-  ];
+      marginTop(-40),
+    ];
 
-  let commandText = (~theme: Theme.t, ~font: UiFont.t) => Style.[
+  let commandText = (~theme: Theme.t, ~font: UiFont.t) =>
+    Style.[
       fontFamily(font.fontFile),
       fontSize(12),
       backgroundColor(theme.background),
       color(theme.editorForeground),
-  ];
-  
-  let controls = Style.[
-    flexGrow(0),
-    flexDirection(`Column),
-    justifyContent(`Center),
-    alignItems(`Center),
-    margin(64),
-  ];
-};
-
-module KeyBindingView = {
+    ];
 
-  module Styles = {
-    let container = Style.[
-      flexDirection(`Row),
+  let controls =
+    Style.[
+      flexGrow(0),
+      flexDirection(`Column),
       justifyContent(`Center),
       alignItems(`Center),
-      height(25),
-      width(300)
+      margin(64),
     ];
-  
-  let commandText = (~theme: Theme.t, ~fontFile, ~fontSize as fs) => Style.[
-      fontFamily(fontFile),
-      fontSize(fs),
-      backgroundColor(theme.background),
-      color(theme.editorForeground),
-  ];
+};
 
-    let spacer = Style.[
-      flexGrow(1)
-    ];
+module KeyBindingView = {
+  module Styles = {
+    let container =
+      Style.[
+        flexDirection(`Row),
+        justifyContent(`Center),
+        alignItems(`Center),
+        height(25),
+        width(300),
+      ];
+
+    let commandText = (~theme: Theme.t, ~fontFile, ~fontSize as fs) =>
+      Style.[
+        fontFamily(fontFile),
+        fontSize(fs),
+        backgroundColor(theme.background),
+        color(theme.editorForeground),
+      ];
```

Comment:
If you remove `Style.` here in favor of `open Style;` at the beginning of the `Styles` module, you don't have to rename `~fontSize`. Instead you can qualify the style property: `Style.fontSize(fontSize)`. Much less awkward IMO.

Reply:
Nice!